man/systemd-udevd.service.xml

```diff
@@ -121,6 +121,17 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--ping-timeout=</option></term>
+        <listitem>
+          <para>Set the number of seconds to wait for daemon to finish initialization.
+          if 0 is specified, then the parent process does not wait and exit immediately
+          after the daemon process is forked.
+          If <option>--daemon</option> is not specified, the specified value is ignored.
+          The default is 5 seconds.</para>
```

Comment:
I think this should be caclled "--daemon-timeout"? It then matches the fact that "--daemon" option is used nicely.